android/src/main/java/com/reactnativecommunity/viewpager/VerticalViewPager.java

```diff
@@ -0,0 +1,109 @@
+package com.reactnativecommunity.viewpager;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.GestureDetector;
+import android.view.MotionEvent;
+import android.view.View;
+
+import android.support.v4.view.ViewPager;
+
+// Vertical ViewPager implement, original code from
+// https://android.googlesource.com/platform/packages/apps/DeskClock/+/master/src/com/android/deskclock/VerticalViewPager.java
+public class VerticalViewPager extends ViewPager {
+    private boolean mVertical = false;
+    private GestureDetector mGestureDetector;
+
+    public VerticalViewPager(Context context) {
+        this(context, null);
+    }
+
+    public VerticalViewPager(Context context, AttributeSet attrs) {
+        super(context, attrs);
+        setOrientation(false);
+    }
+
+    public void setOrientation(boolean vertical) {
+        mVertical = vertical;
+
+        // Make page transit vertical
+        setPageTransformer(true, mVertical ? new VerticalPageTransformer() : null);
+
+        // Get rid of the overscroll drawing that happens on the left and right (the ripple)
+        setOverScrollMode(View.OVER_SCROLL_NEVER);
```

Comment:
This is affecting now the vertical and the horizontal. Can we remove this line? Maybe we can add an extra `prop` if uses wants to remove that overscroll ripple effect.

Reply:
removed